lib/rest_openshift.rb

```diff
@@ -6,7 +6,7 @@ module OpenShift
       extend Helper
 
       def self.populate(path, base_opts, opts)
-        populate_common("/oapi/<oapi_version>", path, base_opts, opts)
+        populate_common("/apis/user.openshift.io/<oapi_version>", path, base_opts, opts)
```

Comment:
`::populate` method is used by all calls. And I believe `user.openshift.io` is only for the user subsystem and will not work for `::list_projects` for example.

I think that we need to drop this openshift/kube api calls separation because now openshift calls are not a separate endpoint, just extensions to kube API. And we can handle all calls in same way.

Reply:
@akostadinov do you mean get ride of `lib/rest_openshift.rb` and merge it into `lib/rest_kubernetes.rb`